### ISSUE TITLE
Failure to pull image of the stack: unify Pulling nog matching manifest for linux/arm/v7 . . 

### ISSUE THREAD

**Issue by pjdw** [ryansch/docker-unifi-rpi#98]
I have the unifi running in a container on a RPI4 which I watch with Diun. It triggered that there is an update. But when deploying my stack I get the above failure.
My stack code is:
```
version: '2'
services:
  unifi:
    image: ryansch/unifi-rpi:latest
    container_name: unifi
    restart: unless-stopped
    network_mode: host
    volumes:
``` 
etc
Is there a way to solve this?

**Comment by ryansch** [comment]
@pjdw We had to drop support for armv7 as there's no longer a mongo that can run on that arch. You'll need to run 64-bit linux on your rpi4.

**Comment by pjdw** [comment]
Thanks! What would be the image line in the stack code
```
   image: ryansch/. . .?
```

**Comment by ryansch** [comment]
Can you paste the output of `uname -a`?

**Comment by pjdw** [comment]
Thats's:
`Linux pi4mk 5.10.92-v7l+ #1514 SMP Mon Jan 17 17:38:03 GMT 2022 armv7l GNU/Linux`

**Comment by ryansch** [comment]
@pjdw Ok that confirms that you're running a 32 bit distro. You'll have to back up your data from that pi and install a 64 bit version of either raspberry pi os or ubuntu.

I personally use ubuntu on all of mine now: https://ubuntu.com/download/raspberry-pi

You can run a desktop but I use ubuntu server.

**Comment by ryansch** [comment]
@pjdw If this is all greek to you, let me know. I could arrange a time to help you on zoom.

**Comment by pjdw** [comment]
That is a very kind offer! First I will try on my one. Good exercise. If I don't manage I will get back.

**Comment by pjdw** [comment]
I managed to get ubuntu server and docker on my Pi. My container looks functional. I can see my devices and the clients on my netwerk, but there are some weird things in my log:


`WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.apache.catalina.loader.WebappClassLoaderBase (file:/usr/lib/unifi/lib/tomcat-embed-core-9.0.59.jar) to field java.io.ObjectStreamClass$Caches.localDescs
WARNING: Please consider reporting this to the maintainers of org.apache.catalina.loader.WebappClassLoaderBase
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release`

Do I have to worry about this; can I solve this?

**Comment by ryansch** [comment]
Ok that's weird. #101 is complaining about the same problem. I have no idea what that is.

**Comment by ryansch** [comment]
@pjdw That warning is resolved as of 7.5.165!

**Comment by pjdw** [comment]
Thanks. I removed the latest tag and used the specific tag. No more warnings but log is a lot longer ;)
